web3-dev/smart-contracts/getting-started.rst

```diff
@@ -97,7 +96,7 @@ Let's go!
 Configure the client
 ~~~~~~~~~~~~~~~~~~~~
 
-Make sure that you have the last version of the Massa node. If not, `install it <https://github.com/massalabs/massa/wiki/install>`_.
+Make sure that you have the last version of the Massa node. If not, `install it <https://docs.massa.net/en/latest/testnet/install.html>`_ and `run it <https://docs.massa.net/en/latest/testnet/running.html>`_.
```

Comment:
These links should probably replaced by relative links (check rst doc if needed/index.rst).

Reply:
I use ref